File: bubble-sort/bubble-sort.rb
def bubble_sort(array)
  i = 0
  while i < array.size - 1
    j = i + 1
    while j < array.size
      array[i], array[j] = array[j], array[i] if array[i] > array[j]
      j = j + 1
    end
    i = i + 1
  end
  array
end

File: solutions/c/bubble-sort.c
#include <stdio.h>

void bubblesort(int *v){

    int i,j,aux;

    int size = sizeof(v) / sizeof(v[0]); // Number of indexes in the V array.

    for (i=0; i < size-1; i++) {
        for(j=i+1; j < size; j++) {
            if(v[i] > v[j]) {
                aux=v[i];
                v[i]=v[j];
                v[j]=aux;
            }
        }
    }
}